src/components/PriceChange.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+import React, { Component } from "react";
+import LText from "./LText";
+import colors from "../colors";
+
+export default class PriceChange extends Component<{
+  before: number,
+  after: number,
+  color?: "western" | "eastern",
```

Comment:
instead of accepting color here, I think you should accept a `locale` prop, and we would use `withLocale()` to inject it. ( https://github.com/LedgerHQ/ledger-wallet-mobile/blob/master/src/components/LocaleContext.js#L60-L72 )

then we basically need to figure out how to infer color from that locale  🤔  . also would this be something in global settings to be able to override it?